server.js

```diff
@@ -3,6 +3,14 @@ const cors = require("cors");
 
 const app = express();
 
+//  app.use(express.json())
+
+const bp = require('body-parser');
```

Comment:
Why are you importing this library? It is not normally included in modern express apps: https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c